### DISCUSSION TITLE
XML syntax highlighting with two color tags

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#3985] [Q&A]
Linux/Debian

I apologize I am relatively new to customizing Geany.
The default syntax highlighting that Geany uses for XML tags is all one color (blue)
I have seen other applications use two colors for these tags: one for the font and another for the Angle Brackets </>
I was wondering if anyone knew of any XML syntax highlighting themes that provided this or the best way to modify the default syntax provided by Geany which I would prefer to do if this is easy enough.

Thank You

**Comment by elextr** [comment]
The highlighting lexer does not distinguish the `<>`s and the tagname, they are the same style so it is not possible for any theme to make them different.

Geany uses the highlighting lexer from the separate Lexilla project so you would have to request a change there.

**Comment by advice2020** [comment]
Thank you
For any users interested, I put in the request but the response did not sound promising that this would be added.
